iris-mpc-bins/ideal_config.toml

```diff
@@ -0,0 +1,15 @@
+graph_size = 1000
+irises_path = "data/store.ndjson"
+layer0_path = "store_ideal_knn_fhd.txt"
+prf_seed = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
+echoice = "NaiveFHD"
+sanity_check = true
+
+[searcher]
+M = [256, 256, 256, 20, 10]
+M_max = [10, 10, 10, 10, 20]
```

Comment:
Why these numbers? Those should be at least equal to `M` and `M_max[0] = 2* M[0]`. See `HnswParams::new`.